src/plugins/condition/Condition.js

```diff
@@ -25,6 +25,7 @@ import uuid from 'uuid';
 import TelemetryCriterion from "./criterion/TelemetryCriterion";
 import { TRIGGER } from "./utils/constants";
 import {computeCondition, computeConditionByLimit} from "./utils/evaluator";
```

Comment:
let's be consistent with spacing inside  {} in imports. I think the more common use is space before and after:
`import { computeCondition, computeConditionByLimit } from "./utils/evaluator";`